flink-table/flink-table-planner-blink/src/main/java/org/apache/flink/table/catalog/CatalogCalciteSchema.java

```diff
@@ -39,10 +39,13 @@
 
 	private final String catalogName;
 	private final Catalog catalog;
+	// Flag that tells if the current planner should work in a batch or streaming mode.
+	private final boolean isBatchMode;
```

Comment:
Could we unify `isBatchMode ` and the following `isStreamingMode` to the same name?